packages/generator-one-app-module/.gitignore

```diff
@@ -0,0 +1,4 @@
+node_modules
```

Comment:
This file can be removed bc this stuff already exists in the top level `.gitignore`.